python/pfs/pipe2d/gitChangelog.py

```diff
@@ -279,47 +299,69 @@ def extractTags(self, url):
 
         Returns
         -------
-        sha_tags : `dict` (`str`: `str`)
-            mapping of commit sha to git tag
+        tag_commit : `dict` (`str`: `list` [`str`])
+            mapping of tag to commit sha
         """
-        sha_tag = {}
+        pattern = re.compile(TAG_REGEX)
+        tag_commit = {}
         pages = self.paginator.pages(url)
         for page in pages:
             for entry in page:
                 sha = entry['commit']['sha']
                 name = entry['name']
-                sha_tag[sha] = name
-        return sha_tag
+                if not pattern.match(name):
+                    logger.debug(f'Tag "{name}" '
+                                 'doesnt match m.n.p or m.n formats. '
+                                 'Skipping.')
```

Comment:
Not necessarily. There are tags other than weekly and releases present, eg ``TEST-PIPE2D-397``, ``v0.9``. It might be better to have a regex that supports both weeklies and release tags, but I intended that as part of a separate ticket.